share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -461,6 +461,28 @@ build_systems-build:
     - artifacts: True
       job: build_systems-generate
 
+###########################################
+# Build tests for different developer tools
+###########################################
+.developer_tools:
+  extends: [ ".linux_x86_64_v3" ]
+  variables:
+    SPACK_CI_STACK_NAME: developer_tools
+
+developer_tools-generate:
+  extends: [ ".developer-tools", ".generate-x86_64"]
```

Comment:
`-` should be a `_`

Reply:
What if we made it standardly `developer-tools` instead to match with other stack conventions?

Reply:
Darn. I had made that change but must have lost it in the rebase where I removed a lot of intermediate commits. Will fix.